vita3k/util/src/util.cpp

```diff
@@ -267,6 +267,47 @@ void float_to_half(const float *src, std::uint16_t *dest, const int total) {
     }
 }
 
+#ifdef __AVX2__
```

Comment:
Why AVX2? It should be AVX + F16C

Reply:
I don't know how to check exactly AVX + F16C at compile-time. common compiler macros are __AVX__ and __AVX2__  I've found some checks but they were compiler-specific.
As sayed on wikipedia Intel Sandy Bridge has AVX but does not have F16C, so checking __AVX__ is not enough.

Reply:
`#if defined(__AVX__) && defined(__F16C__)` does not work?

Reply:
I can't check, but as told here https://git.sailfishos.org/mer-core/qtbase/commit/280e321e52fd4e86545f3f0d4bd4e047786a897e and there https://stackoverflow.com/questions/16348909/how-do-i-know-if-i-can-compile-with-fma-instruction-sets "Unfortunately MSVC doesn't define any instruction set specific macros other than __AVX__ and __AVX2__"

Reply:
maybe will work #if (defined(__AVX__) && defined(__F16C__)) || defined(__AVX2__)